ui/src/dashboards/actions/index.js

```diff
@@ -149,7 +171,7 @@ const templates = [
 export const getDashboardsAsync = dashboardID => async dispatch => {
   try {
     const {data: {dashboards}} = await getDashboardsAJAX()
-    const stubbedDashboards = dashboards.map(d => ({...d, templates}))
```

Comment:
should be `templates`

https://github.com/influxdata/chronograf/blob/df6b90cc03b8cd1d84b6ad2ebab73390aaac4887/server/dashboards.go#L20